app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -316,7 +316,7 @@ export const ContentForm: React.FC<IContentFormProps> = ({
             }
           >
             {(props: FormikProps<IContentForm>) => (
-              <Col>
+              <Col style={{ display: 'block' }}>
```

Comment:
Do not do inline styles like this.  Place it in the styled-component like the majority of our styling.

Reply:
Updated, thanks